pkg/network/node/healthcheck.go

```diff
@@ -67,8 +67,8 @@ func runOVSHealthCheck(network, addr string, healthFn func() bool) {
 				glog.V(2).Infof("SDN healthcheck unable to ping OVS server: %v", err)
 				return false, nil
 			}
-			if !healthFn() {
-				return false, fmt.Errorf("OVS health check failed")
+			if err := healthFn(); err != nil {
+				return false, fmt.Errorf("OVS health check failed, %v", err)
```

Comment:
How about: "OVS health check failed: %v"

Reply:
you're right, i'll change it